packages/components/Notification/src/NotificationTokens.ts

```diff
@@ -0,0 +1,22 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { NotificationTokens } from './Notification.types';
+
+export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme> = (t: Theme) =>
+  ({
+    backgroundColor: 'skyblue',
+    color: t.colors.bodyText,
+    borderRadius: 12,
+    padding: 16,
+    neutral: {
+      backgroundColor: 'lightgrey',
```

Comment:
Out of curiosity is there a plan to use design token driven values in the future?

I suppose this is meant for iOS so the issue is that there aren't any tokens to use yet?

Reply:
The plan is to follow the FluentUI Apple control, which is tokenized, refers to some global/alias tokens, but isn't hooked up to the token-pipeline yet. I'm not familiar with the timeline to hook up FluentUI Apple to the token-pipeline, or stand up the iOS design tokens package. Because standing up the iOS design tokens package is out of scope for this project, we'll probably have to copy / hardcode most of the default tokens. I think there are some cases where we can refer to design tokens right now, since it looks like the FluentUI Apple control does. 

If you're curious, the WIP FluentUI Apple control's token definitions are here: https://github.com/microsoft/fluentui-apple/blob/fluent2-tokens/ios/FluentUI/Vnext/Notification/NotificationTokens.swift
You can see line 65 refers to globalTokens.brandColor.tint40, and line 156-160 refers to aliasTokens.shadow.shadow16

Reply:
Sounds good, thanks!

Reply:
Ha, we both commented at the same time :D